src/pytorch_lightning/loggers/csv_logs.py

```diff
@@ -69,7 +69,7 @@ def log_hparams(self, params: Dict[str, Any]) -> None:
     def log_metrics(self, metrics_dict: Dict[str, float], step: Optional[int] = None) -> None:
         """Record metrics."""
 
-        def _handle_value(value):
+        def _handle_value(value: Union[Tensor, Any]) -> Union[Tensor, Any]:
```

Comment:
the return type can't be Tensor, because of `.item()`

Reply:
updated in https://github.com/Lightning-AI/lightning/pull/13538/commits/1582c63d7704079965c1f95466a9ecfdb9d1a389